test/tools/Modules/HelpersCommon/HelpersCommon.psm1

```diff
@@ -219,22 +219,21 @@ function Send-VstsLogFile {
         }
     }
 
+    $newName = ([System.Io.Path]::GetRandomFileName() + "-$LogName.txt")
     if($Contents)
     {
-        $logFile = Join-Path -Path $logFolder -ChildPath ([System.Io.Path]::GetRandomFileName() + "-$LogName.txt")
-        $name = Split-Path -Leaf -Path $logFile
+        $logFile = Join-Path -Path $logFolder -ChildPath $newName
 
         $Contents | Out-File -path $logFile -Encoding ascii
     }
     else
     {
-        $name = Split-Path -Leaf -Path $path
-        $logFile = Join-Path -Path $logFolder -ChildPath ([System.Io.Path]::GetRandomFileName() + '-' + $name)
+        $logFile = Join-Path -Path $logFolder -ChildPath $newName
```

Comment:
Why was this change needed?  I the old code, the two paths would result in different random files, but now would have the same.

Reply:
The code was changing the name when uploading but not logging how it changed the name.  I just changed it to log what it changed the name to.